epochconverter/convert_sdhm.ts

```diff
@@ -0,0 +1,12 @@
+import {element, by, ElementFinder} from 'protractor';
+
+export class SEC {
+	public convert_sdhm: ElementFinder = element(by.id("tc")).element(by.id("scinput"));
+	public convert_sdhm_button: ElementFinder = element(by.id("tc")).element(by.buttonText("Seconds to days, hours, minutes"));
+
+	constructor(seconds:string){
```

Comment:
Keine Logik im Constructor -> Auslagern in eigene Funktion

Reply:
Ist entfernt worden